action.yml

```diff
@@ -33,23 +30,67 @@ runs:
     - name: Setup Node
       uses: actions/setup-node@v3
       with:
-          node-version: 'lts/*'
-          cache: 'npm'
+        node-version: 'lts/*'
 
-    - name: Create configuration file
+    - name: Install wp-env
+      shell: bash
+      run: npm -g i @wordpress/env
+
+    # Although @wordpress/env ships with a default configuration, a missing or
```

Comment:
I don't think a missing `.wp-env.override.json` file will throw an error. You could use only the `.wp-env.json` if you wanted.

Reply:
This Action assumes no `.wp-env.json` is present, although it is possible to include one in the project. We likely need to add some sort of documentation in the README.md that talks about the usage of a `wp-env.json` within the repo, and how this overrides that config.

Otherwise, if the project that uses this action does _not_ have a `wp-env.json`, this will fail, which is why we have that note. Let me consider how we might rework, as well as add some additional documentation to the README.md.

Reply:
I added some information regarding `wp-env.json` to the README.md.

Reply:
@kopepasah I'm surprised this fails if the `.wp-env.json` file is missing. Was there an error describing why?